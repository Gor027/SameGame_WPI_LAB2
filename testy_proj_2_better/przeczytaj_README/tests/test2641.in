!!!
  !c!c !c !!   c!!cc!!! !c!!c!!! cc! c!cc!c  !cc! !! ! 
c ! !c! 
     !c!cc!cc ccc !
c !!! !!   cc!!cc cc c!! cc!cc cccccccc!c!c c ! ccc!cc!!!cc!!!ccc  cc! cc
c  cc! c c !   !   !!c! !   c! c!c cc !c! c c! !  c ! c!cc   cc !
cc!! c!! c! c c!c  ! ccccc  ! !c!!c cc!c!!!  c c! ! c !c  c!c!cc
!c  !  c cc!cc!! !cc  c !!c
