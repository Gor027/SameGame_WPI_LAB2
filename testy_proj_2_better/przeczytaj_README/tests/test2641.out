       !       c
! ! !c !   !   c   c  !
c !c!c!! c!cc cc! cc !! !c  c  ! c    !cc c      !
c!!!!!cc c!cc !!c !! c! !c! c! cccc!ccc!c!c ! c! c c !!     !c
cc!cc!!!cc!!c!c!c!c!!cc!ccc!cc!c!ccccc!c!!cc!c!c c!!!c!!cc!!cccc
cc!!!c!cccc!cc!!c!cc!ccc!!c!!!!c!!cccc!c!!!cc!c!c!!cc!cccc!c!cc!cc!cc
